tests/e2e-pw/utils/navigation/navigation.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { PlaywrightTestArgs } from '@playwright/test';
+import { BlockData } from '@woocommerce/e2e-types';
+
+/**
+ * Closes any modals in the editor if they are open.
+ */
+export const closeModalIfExists = async (
+	page: PlaywrightTestArgs[ 'page' ]
+) => {
+	// The modal close button can have different aria-labels, depending on the version of Gutenberg/WP.
```

Comment:
Good approach. I wonder how it could be a more scalable approach 🤔 . I will think about it when I will work on https://github.com/woocommerce/woocommerce-blocks/issues/9003